client/client.go

```diff
@@ -711,7 +714,13 @@ type ProgressHandler func(*wamp.Result)
 // IMPORTANT: If the context has a timeout, then the amount of time needs to be
 // sufficient for the caller to receive all progressive results as well as the
 // final result.
-func (c *Client) Call(ctx context.Context, procedure string, options wamp.Dict, args wamp.List, kwargs wamp.Dict, progcb ProgressHandler) (*wamp.Result, error) {
+func (c *Client) Call(
+	ctx context.Context,
+	procedure string,
+	options wamp.Dict,
+	args wamp.List,
+	kwargs wamp.Dict,
+	progcb ProgressHandler) (*wamp.Result, error) {
```

Comment:
I prefer not to put each argument on a separate line. It leads to different contributors making different choices about when to split a function args to separate lines, and can make the code look inconsistent. I also prefer to keep more aligned with the golang stdlib code.

I think the long lines also act as a flag telling us that function needs to be examined, and a better interface is probably needed.

I would prefer to keep the func signature on one line, unless you strongly feel this greatly improves readability. Either way, in a future version of nexus we can probably com up with a better interface for calling functions like this.